src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -236,15 +236,15 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     {
       if (macroSubstitute(&payload, httpInfo.payload, ce) == false)
       {
-        LM_W(("Bad Input (not sending NotifyContextRequest: payload too large)"));
+        LM_W(("Runtime Error (not sending NotifyContextRequest: payload too large)"));
         return paramsV;  // empty vector
       }
 
       char* pload  = curl_unescape(payload.c_str(), payload.length());
       payload      = std::string(pload);
       renderFormat = NGSI_V2_CUSTOM;
       mimeType     = "text/plain";  // May be overridden by 'Content-Type' in 'headers'
-      free(pload);
+      curl_free(pload);
```

Comment:
I have take advantage of this PR to do this little fix we recently discover.

NTC (just informative)